test/e2e/tcp/connectivity.go

```diff
@@ -33,7 +33,10 @@ type EndpointType int
 const (
 	PodIP EndpointType = iota
 	ServiceIP
+	// TODO: Remove GlobalIP once all consumer code switches to GlobalServiceIP
 	GlobalIP
+	GlobalPodIP
+	GlobalServiceIP = GlobalIP
```

Comment:
Neither GlobalIP I see.

I think it's OK to break the code when somebody updates to the newer version of shipyard, we can update it to make an explicit choice of GlobalPodIP (let's add it only once implemented) or GlobalServiceIP.

Reply:
GlobalIP is atleast used in Submariner e2e tests. If we remove the GlobalIP constant, it might break the dependent components. The idea was to have it in this manner until we migrate all the dependent components and remove it once all are updated.

Reply:
I think Miguel’s point is that the upgrade won’t happen automatically, we’ll need to bump `go.mod` in the projects using this, and while we’re at it we can fix any breakage.

Reply:
Yeah, that's it, at that point we are good to change the constant, and no risk of leaving an old constant lying around.

Reply:
Okay, in that case, we can remove it now itself.